presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizersFactory.java

```diff
@@ -87,6 +88,8 @@ public PlanOptimizersFactory(Metadata metadata, SqlParser sqlParser, FeaturesCon
                 new IndexJoinOptimizer(metadata), // Run this after projections and filters have been fully simplified and pushed down
                 new CountConstantOptimizer(),
                 new WindowFilterPushDown(metadata), // This must run after PredicatePushDown and LimitPushDown so that it squashes any successive filter nodes and limits
+                new MergeIdenticalWindows(),
+                new PruneUnreferencedOutputs(),
```

Comment:
Guidance on whether this is needed or not from somebody with broader optimizer experience would be appreciated.